scripts/hak/hakEnv.ts

```diff
@@ -114,4 +114,12 @@ export default class HakEnv {
     getNodeModuleBin(name: string): string {
         return path.join(this.projectRoot, 'node_modules', '.bin', name);
     }
+
+    wantsStaticSqlCipherUnix(): boolean {
```

Comment:
upcoming ts style guide won't like these implicit member access levels (`public`/`private`/etc)

Reply:
Be glad this script isn't Javascript :P

Reply:
🥲